collada_urdf_jsk_patch is failing to build on indigo armhf and i386

http://build.ros.org/view/Ibin_arm_uThf/job/Ibin_arm_uThf__collada_urdf_jsk_patch__ubuntu_trusty_armhf__binary/38/console
http://build.ros.org/view/Ibin_uT32/job/Ibin_uT32__collada_urdf_jsk_patch__ubuntu_trusty_i386__binary/28/console

```
Cloning into 'build/robot_model/src'...
cd build/robot_model/src && git checkout indigo-devel
Switched to a new branch 'indigo-devel'
Branch indigo-devel set up to track remote branch indigo-devel from origin.
touch rospack_nosubdirs
cd build/robot_model/src && git reset --hard
HEAD is now at aa25e71 Strip collada indigo (#197)
patch -d build/robot_model/src -p1 < /tmp/binarydeb/ros-indigo-collada-urdf-jsk-patch-2.0.19/use_assimp_devel.patch &&  echo patched
can't find file to patch at input line 5
Perhaps you used the wrong -p or --strip option?
The text leading up to this was:
--------------------------
|diff --git a/collada_urdf/CMakeLists.txt b/collada_urdf/CMakeLists.txt
|index 524f2e9..6bd3707 100644
|--- a/collada_urdf/CMakeLists.txt
|+++ b/collada_urdf/CMakeLists.txt
--------------------------
File to patch: 
Skip this patch? [y] 
Skipping patch.
1 out of 1 hunk ignored
can't find file to patch at input line 25
Perhaps you used the wrong -p or --strip option?
The text leading up to this was:
--------------------------
|diff --git a/collada_urdf/src/collada_to_urdf.cpp b/collada_urdf/src/collada_to_urdf.cpp
|index 616f58a..d4bbf2a 100644
|--- a/collada_urdf/src/collada_to_urdf.cpp
|+++ b/collada_urdf/src/collada_to_urdf.cpp
--------------------------
File to patch: 
Skip this patch? [y] 
Skipping patch.
1 out of 1 hunk ignored
can't find file to patch at input line 48
Perhaps you used the wrong -p or --strip option?
The text leading up to this was:
--------------------------
|diff --git a/collada_urdf/src/collada_urdf.cpp b/collada_urdf/src/collada_urdf.cpp
|index 23be87c..1aeac58 100644
|--- a/collada_urdf/src/collada_urdf.cpp
|+++ b/collada_urdf/src/collada_urdf.cpp
--------------------------
File to patch: 
Skip this patch? [y] 
Skipping patch.
1 out of 1 hunk ignored
make[5]: *** [patched] Error 1
make[5]: Leaving directory `/tmp/binarydeb/ros-indigo-collada-urdf-jsk-patch-2.0.19/obj-i686-linux-gnu'
make[4]: *** [devel/lib/collada_urdf_jsk_patch/urdf_to_collada] Error 2
make[4]: Leaving directory `/tmp/binarydeb/ros-indigo-collada-urdf-jsk-patch-2.0.19/obj-i686-linux-gnu'
make[3]: *** [CMakeFiles/urdf_to_collada.dir/all] Error 2
make[3]: Leaving directory `/tmp/binarydeb/ros-indigo-collada-urdf-jsk-patch-2.0.19/obj-i686-linux-gnu'
make[2]: *** [all] Error 2
make[2]: Leaving directory `/tmp/binarydeb/ros-indigo-collada-urdf-jsk-patch-2.0.19/obj-i686-linux-gnu'
dh_auto_build: make -j1 returned exit code 2
make[1]: *** [override_dh_auto_build] Error 2
make[1]: Leaving directory `/tmp/binarydeb/ros-indigo-collada-urdf-jsk-patch-2.0.19'
make: *** [build] Error 2
dpkg-buildpackage: error: debian/rules build gave error exit status 2
E: Building failed
Traceback (most recent call last):
  File "/tmp/ros_buildfarm/ros_buildfarm/binarydeb_job.py", line 133, in build_binarydeb
    subprocess.check_call(cmd, cwd=source_dir)
  File "/usr/lib/python3.4/subprocess.py", line 561, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['apt-src', 'build', 'ros-indigo-collada-urdf-jsk-patch']' returned non-zero exit status 1

--------------------------------------------------------------------------------------------------
`apt-src build ros-indigo-collada-urdf-jsk-patch` failed.
This is usually because of an error building the package.
The traceback from this failure (just above) is printed for completeness, but you can ignore it.
You should look above `E: Building failed` in the build log for the actual cause of the failure.
--------------------------------------------------------------------------------------------------

# END SUBSECTION
Build step 'Execute shell' marked build as failure
$ ssh-agent -k
unset SSH_AUTH_SOCK;
unset SSH_AGENT_PID;
echo Agent pid 2507 killed;
[ssh-agent] Stopped.
[description-setter] Could not determine description.
Sending e-mails to: ros-buildfarm-indigo@googlegroups.com tfoote+buildfarm@osrfoundation.org k-okada@jsk.t.u-tokyo.ac.jp youhei@jsk.t.u-tokyo.ac.jp
Warning: you have no plugins providing access control for builds, so falling back to legacy behavior of permitting any downstream builds to be triggered
Finished: FAILURE
```

I believe that this has to do with the updated upstream release: https://github.com/ros/rosdistro/pull/14897

--- a/jsk_ros_patch/collada_urdf_jsk_patch/CMakeLists.txt
+++ b/jsk_ros_patch/collada_urdf_jsk_patch/CMakeLists.txt
@@ -3,9 +3,9 @@ cmake_minimum_required(VERSION 2.8.3)
 project(collada_urdf_jsk_patch)
 
 if(("$ENV{ROS_DISTRO}" STREQUAL "groovy") OR ("$ENV{ROS_DISTRO}" STREQUAL "hydro"))
-  set(SOURCE_DISTRO hydro)
+  set(SOURCE_DISTRO hydro-devel)
 elseif ("$ENV{ROS_DISTRO}" STREQUAL "indigo" OR "$ENV{ROS_DISTRO}" STRGREATER "indigo")
-  set(SOURCE_DISTRO indigo)
+  set(SOURCE_DISTRO bd4fc369d56eaa0c31d8cb17677e00b9d9685de6) # 1.11.13, before strip indigo
 endif()
 if ("$ENV{ROS_DISTRO}" STREQUAL "kinetic")
     set(CXXFLAGS CXXFLAGS=-std=gnu++11)

--- a/jsk_ros_patch/collada_urdf_jsk_patch/Makefile
+++ b/jsk_ros_patch/collada_urdf_jsk_patch/Makefile
@@ -3,7 +3,7 @@ all: urdf_to_collada
 
 GIT_DIR = build/robot_model/src
 GIT_URL = git://github.com/ros/robot_model.git
-GIT_REVISION = ${SOURCE_DISTRO}-devel
+GIT_REVISION = ${SOURCE_DISTRO}
 PATCH_DIR = $(CURDIR)
 GIT_PATCH = ${PATCH_DIR}/use_assimp_devel.patch
 BUILD_BIN_DIR  = build/robot_model/devel/lib/collada_urdf

